pyfar/classes/audio.py

```diff
@@ -373,6 +373,7 @@ def time(self, value):
 
     @property
     def complex(self):
+        """Return or set the flag indicating if the data is complex."""
```

Comment:
Maybe more explicit: `"""Return or set the flag indicating if the time data is complex."""`. The property does not exist for Frequency data.